hyperspy/tests/signal/test_1D_tools.py

```diff
@@ -81,22 +82,6 @@ def test_align(self):
         nt.assert_true((s.data[:, -1] == 2).all())
         nt.assert_true((s.data[:, 0] == 2).all())
         # Check that the calibration is correct
-        nt.assert_equal(s.axes_manager._axes[1].offset, self.new_offset)
```

Comment:
that's just git / github trying to lump changes. The lines are still there in the new file, but this diff shows them as being after the removed lines.... (in the old files these lines are 100-102)